acceptance/tests/apply_nonroot.rb

```diff
@@ -63,7 +54,7 @@
       result = json.select { |n| n['node'] == host }
       assert_equal('failure', result[0]['status'],
                    "The task did not fail on #{host}")
-      assert_match(/Permission denied/, result[0]['result']['_error']['msg'])
```

Comment:
This behavior is weird, going to look into it a bit more. Presumably Puppet is returning a different exit code based on `--tty`.

Reply:
Message is in `_output` because it appears on stdout rather than stderr with `--tty`. We should probably discourage using `--tty` generally.